wrappers/python/btllib.py

```diff
@@ -1,12 +1,107 @@
-# This file was automatically generated by SWIG (https://www.swig.org).
-# Version 4.1.1
+# This file was automatically generated by SWIG (http://www.swig.org).
+# Version 4.0.2
```

Comment:
Please upgrade SWIG to the latest version and regenerate the wrappers

Reply:
Are you using pip to install swig. Conda swig hasn't been updated.

Reply:
Conda-forge swig is 4.1.1: https://anaconda.org/conda-forge/swig

Reply:
done